nilrt_snac/_Configs/_CryptsetupConfig.py

```diff
@@ -0,0 +1,23 @@
+import argparse
+
+from nilrt_snac._Configs._BaseConfig import _BaseConfig
+
+from nilrt_snac import logger
+from nilrt_snac.OpkgHelper import opkg_helper
+
+
+class _CryptsetupConfig(_BaseConfig):
```

Comment:
`_CryptSetupConfig`?

Also
* `_NIAuthConfig`
* `_NTPConfig`
* `_OPKGConfig`
* `_USBGuardConfig`
* `_WIFIConfig`

Reply:
All are renamed

Reply:
These are still using lower-case letters in Patchrev 2.